### ISSUE TITLE
Predicitve Model

### ISSUE THREAD

**Issue by leonsuetfeld** [leonsuetfeld/moraldm#5]
Ich denke wir sollten im Paper diese Eigenheit ansprechen, dass unsere Main Effects systematisch anwachsen, wenn wir als zweites Level die sn_idx mit dazu nehmen. Angenommen, wir würden im Auto nachher ein solches Model laufen zur Entscheidungsfindung in solchen Situationen laufen lassen, würden wir ja em ehesten ein single level model nehmen, das dann entsprechend kleinere Main Effects hätte.

Das wiederum gibt uns dann auch die Möglichkeit, eine nicht-redundante vereinfachte Figure zu bauen, die das Model zeigen könnte, wie man es im Auto letztendlich einsetzen könnte.

Meinung hierzu? :)

**Comment by behinger** [comment]
So ein model wird niemals im Auto eingesetzt werden, ich glaube das ist
nicht so wichtig. Und wenn es eingesetzt werden sollte, würde ich
aufjedenfall das Multi-Level model und nicht das single level model
benutzen :)



Am 28.09.2018 um 14:21 schrieb Leon Suetfeld:

**Comment by leonsuetfeld** [comment]
Doch, sowas könnte man sehr gut einsetzen (natürlich mit anderen Prädiktoren und Werten). Und ich denke auch, dass man wenn dann am ehesten ein einfach zu verstehendes und zu implementierendes single-level model verwenden würde.

**Comment by behinger** [comment]
Ich glaube nicht dass man ein probabilistisches Model einsetzen würde. Sowieso glaube ich nicht, dass es so ein model jemands geben wird (aber das ist besides the point). 

Unser model ist auch deskriptiv, es ist überhaupt nicht für predictions optimiert - ich finde das geht weit über die Studie hinaus so etwas zu designen und müsste in einem eigenen Paper geschehen. Alleine so etwas vorzuschlagen hat so viele rießiege ethische problematiken, da weiß ich garnicht wo ich anfangen soll ;-)

**Comment by leonsuetfeld** [comment]
Es geht darum, ein Model vorzuschlagen, das zur Entscheidungsfindung in Dilemma-Situationen eingesetzt werden kann. Es geht hier nicht darum, ein probabilistisches Model im Sinne von non-deterministisch zu verwenden. Aber es geht darum, ein Model zu finden, das, gegeben einer Reihe von Optionen in einer kritischen Situation, eine Abwägung macht, welche dieser Optionen eine Loss-Function minimiert.

Die ethischen Entscheidungen, um die es hier geht, sind eben meiner Meinung nach nicht sauber mit Entscheidungsbäumen zu lösen.

Was ich im Paper kurz diskutieren möchte, ist, wie eine solche Loss-Function grob aussehen könnte, und was man dafür bräuchte.

**Comment by behinger** [comment]
Ok verstehe ich besser. Aber das model ist probabilistisch, weil du ja eine wahrscheinlichkeit vorhersagst und die ist nicht diskret.

Ich würde die kiste zwischen single & multi-level model nicht aufbrechen. Ich traue dem single-level model deutlich weniger als dem multi-level model. Wenn du die entscheidung diskretisieren würdest (also nicht probabilistisch), würde bei den modellen sowieso dasselbe rauskommen denke ich.
